pkg/local_object_storage/metabase/select.go

```diff
@@ -473,6 +536,66 @@ func (db *DB) matchSlowFilters(tx *bbolt.Tx, addr oid.Address, f object.SearchFi
 	}
 
 	for i := range f {
+		switch op := f[i].Operation(); op {
+		case object.MatchNumGT, object.MatchNumGE, object.MatchNumLT, object.MatchNumLE:
+			attr := f[i].Header()
+			if attr != object.FilterCreationEpoch && attr != object.FilterPayloadSize {
+				break
+			}
+
+			// filter by creation epoch or payload size, both uint64
+			filterVal := f[i].Value()
+			if len(filterVal) == 0 {
+				return false
+			}
+
+			if filterVal[0] == '-' {
+				if op == object.MatchNumLT || op == object.MatchNumLE {
+					return false
+				}
+				continue
+			}
+
+			if len(filterVal) > 20 { // max uint64 strlen
```

Comment:
what if it is 20 len but bigger than uint64? do we return an error for this? even if that is a correct comparison?

Reply:
that was a mistake, fixed